wremnants/datasets/datagroups.py

```diff
@@ -198,7 +198,7 @@ def excludeGroups(self, excludes):
             logger.warning(f"Excluded all groups using '{excludes}'. Continue without any group.")
 
     def set_histselectors(self, 
-                          group_names, histToRead="nominal", fake_processes=None, mode="extended2D", smoothen=False, smoothingOrderFakerate=1, simultaneousABCD=False, forceGlobalScaleFakes=None, **kwargs
+                          group_names, histToRead="nominal", fake_processes=None, mode="extended1D", smoothen=False, smoothingOrderFakerate=2, simultaneousABCD=False, forceGlobalScaleFakes=None, **kwargs
```

Comment:
shouldn't this also be smoothen=True ?

Reply:
Technically yes, it depends on whether we only want the default order to be 2 when the smoothing is requested, or set the smoothing by default as well. I can go for the latter, trying to change other scripts to maintain the older behaviour when the binned version was used (without assuming the default values, since now they would change)

Reply:
The default for the analysis should now be extended 1d with smoothing and order 2

This need to be the default behaviour for all scripts, otherwise we will be very likely to end up with inconsistent plots and confusion.